packages/components/src/index.ts

```diff
@@ -33,7 +33,9 @@ export {
 } from './autocomplete';
 export { default as BaseControl, useBaseControlProps } from './base-control';
 export {
+	/** @deprecated Import `BorderBoxControl` instead. */
 	BorderBoxControl as __experimentalBorderBoxControl,
+	BorderBoxControl,
 	hasSplitBorders as __experimentalHasSplitBorders,
 	isDefinedBorder as __experimentalIsDefinedBorder,
 	isEmptyBorder as __experimentalIsEmptyBorder,
```

Comment:
What about those experimental utils? They are part of the component's API, but they remain experimental. Should we stabilize them as well?

Reply:
This is something I want to address in a follow-up. I already gave it a few iterations some time ago, and even paired on it with Marcelo over a donut chat which resulted in a draft PR I plan on reviving. There was some consensus about not exporting these from the component package, though the details remain to be discussed and iterated on.

I don't think it should block stabilization of this component though.

Reply:
> I don't think it should block stabilization of this component though.

Why, though, aren't those part of the component's public API? I'd expect it would be the other way around, and we'd figure these out first, especially if we plan not to export them from the components package / deprecate them.

Reply:
Related: https://github.com/WordPress/gutenberg/issues/61135

Reply:
> Why, though, aren't those part of the component's public API? I'd expect it would be the other way around, and we'd figure these out first, especially if we plan not to export them from the components package / deprecate them.

I'm not strongly opposed that that, but to answer your question, my opinion is: no, they are not part of the component's API. I think these utils shouldn't have been exported here in the first place, and it's just an odd occurrence of misplaced utils. They're definitely _related_ to the component, in that they apply to borders, but the component is its own thing.

So, the way I see it: stabilizing the component, moving the utilities somewhere else - these are separate tasks. Should they be tackled somewhat close to each other? I think so. Should one block the other? I think not.

That's just my 2 cents though. I'm good to go with another approach if someone feels more strongly about it.

Reply:
Fair enough. Let's keep the utils a separate problem and solve it as part of #61135 👍